libvirt/tests/src/virsh_cmd/domain/virsh_migrate.py

```
@@ -1112,7 +1112,7 @@ def check_vm_state_dest():
                 obj_migration.do_migration(vms, src_uri, dest_uri, "orderly",
                                            options=migrate_options,
                                            thread_timeout=postcopy_timeout,
-                                           ignore_status=False,
+                                           ignore_status=True,
```

The change will allow silent failure. 
Could you describe a little more about how the change match with  the PR's purpose  "The error message was unclear if migration failed, so update to
provide a proper error message." ?

---

No, this pr doesn't allow silent failure. From the test result I provided, you can see, this case still failed. Checking migration results is a mandatory step after migration.The difference is that the error message has been updated with this change.